src/emulator/modules/SceProcessmgr/SceProcessmgr.cpp

```diff
@@ -44,8 +45,18 @@ EXPORT(int, sceKernelLibcClock) {
     return unimplemented("sceKernelLibcClock");
 }
 
-EXPORT(int, sceKernelLibcGettimeofday) {
-    return unimplemented("sceKernelLibcGettimeofday");
+EXPORT(int, sceKernelLibcGettimeofday, Ptr<VitaTimeval> timeAddr, Ptr<Address> tzAddr) {
+    if (timeAddr)
+    {
+        auto* tv = timeAddr.get(host.mem);
+
+        const auto ticks = rtc_get_ticks(host);
+
+        tv->tv_sec = ticks / 1'000'000;
```

Comment:
I think we may have a define for 1000000 somewhere. `VITA_CLOCK_PER_SEC` or something like it?

Reply:
You're right, forgot about it.